services/market/shared/market/sources/bittrex.js

```diff
@@ -37,12 +37,13 @@ const symbolMap = {
 const fetchAllMarketTickers = async () => {
 	try {
 		const response = await requestLib(`${apiEndpoint}/markets/tickers`);
+		if (response === undefined) throw new Error('Data from Bittrex is unavailable');
```

Comment:
Does it mean the error message with a stack trace will be displayed when Bittrex is unavailable? I think it deserves only a mention in the custom warning message.

Reply:
@nagdahimanshu Please make the changes within the jobs as we discussed.

Reply:
Done!